### PR TITLE
chore(cargo): add metadata

### PR DESCRIPTION
**Pull request by storopoli**
Feel free to suggest edits.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -3,6 +3,22 @@ name = "rmathlib"
 description = "A Rust Port of R's C Library of Special Functions"
 version = "1.0.0"
 edition = "2021"
+authors = [
+  "Rik Huijzer <rik@huijzer.xyz>",
```
Comment (rikhuijzer) [inline Cargo.toml:7]:
```suggestion
  "Rik Huijzer <github@huijzer.xyz>",
```

File: Cargo.toml [hunk 2]
```diff
@@ -3,6 +3,22 @@ name = "rmathlib"
...
+  "Jose Storopoli <jose@storopoli.io>",
+]
+license = "GPL 3.0"
+categories = ["science"]
```
Comment (rikhuijzer) [inline Cargo.toml:11]:
```suggestion
categories = ["science", "mathematics", "algorithms"]
```
mathematics is obvious. Algorithms maybe too?

File: Cargo.toml [hunk 3]
```diff
@@ -3,6 +3,22 @@ name = "rmathlib"
 description = "A Rust Port of R's C Library of Special Functions"
 version = "1.0.0"
 edition = "2021"
+authors = [
```
Comment (rikhuijzer) [inline Cargo.toml:6]: Maybe authors "The PowerAnalyses.org contributors" like https://github.com/rust-random/rand/blob/master/Cargo.toml

File: Cargo.toml [hunk 4]
```diff
@@ -3,6 +3,22 @@ name = "rmathlib"
 description = "A Rust Port of R's C Library of Special Functions"
 version = "1.0.0"
 edition = "2021"
```
Comment (rikhuijzer) [inline Cargo.toml:5]:
```suggestion
edition = "2021"
documentation = "https://rmathlib.poweranalyses.org/rmathlib"
```

File: Cargo.toml [hunk 5]
```diff
@@ -3,6 +3,22 @@ name = "rmathlib"
 description = "A Rust Port of R's C Library of Special Functions"
 version = "1.0.0"
```
Comment (rikhuijzer) [inline Cargo.toml:4]:
```suggestion
version = "1.0.0"
include = ["src/", "LICENSE", "README.md"]
```
To ensure C is not bundled

File: Cargo.toml (hunk 3 above)
Comment (storopoli) [reply Cargo.toml:6]: Following the `libm`'s one: https://github.com/rust-lang/libm/blob/master/Cargo.toml
It is maintained by `rust-lang` with 39 contributors but somehow has a single author.

File: Cargo.toml (hunk 4 above)
Comment (storopoli) [reply Cargo.toml:5]: Dunno, aren't we publishing this? Then the link would change to docs.rs/rmathlib in the near future.

**Review by rikhuijzer — APPROVED** [review]

**Pull request merged**